src/lib.rs

```diff
@@ -150,6 +150,10 @@ mod checksum;
 pub mod closure;
 mod enums;
 mod file_error;
+#[cfg(not(windows))]
+mod functions;
+#[cfg(not(windows))]
+pub use functions::*;
```

Comment:
This is a potential footgun. Someone who might want to add a new function there later might miss this and then it's only available for non-Windows :)

Reply:
Ok.